<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/icon/png图片/2.png);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件（&amp;F）</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_N"/>
    <addaction name="action_O"/>
    <addaction name="separator"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_P"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑（&amp;E)</string>
    </property>
    <addaction name="fontAction"/>
    <addaction name="colorAction"/>
    <addaction name="dateAction"/>
    <addaction name="separator"/>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="copyAction"/>
    <addaction name="cutAction"/>
    <addaction name="pastAction"/>
    <addaction name="selectAllAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助($H)</string>
    </property>
    <addaction name="aboutSoftwareAction"/>
    <addaction name="aboutWebAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_N"/>
   <addaction name="action_O"/>
   <addaction name="action_S"/>
   <addaction name="separator"/>
   <addaction name="copyAction"/>
   <addaction name="fontAction"/>
   <addaction name="colorAction"/>
   <addaction name="pastAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_N">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/png图片/1.png</normaloff>:/icon/png图片/1.png</iconset>
   </property>
   <property name="text">
    <string>新建（&amp;N)</string>
   </property>
  </action>
  <action name="action_O">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/png图片/2.png</normaloff>:/icon/png图片/2.png</iconset>
   </property>
   <property name="text">
    <string>打开（&amp;O）</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/png图片/3.png</normaloff>:/icon/png图片/3.png</iconset>
   </property>
   <property name="text">
    <string>保存（&amp;S)</string>
   </property>
  </action>
  <action name="action_A">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/png图片/4.png</normaloff>:/icon/png图片/4.png</iconset>
   </property>
   <property name="text">
    <string>另存为（&amp;A)</string>
   </property>
  </action>
  <action name="action_P">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/png图片/5.png</normaloff>:/icon/png图片/5.png</iconset>
   </property>
   <property name="text">
    <string>打印(&amp;P)</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/png图片/7.png</normaloff>:/icon/png图片/7.png</iconset>
   </property>
   <property name="text">
    <string>退出（&amp;)</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/png图片/8.png</normaloff>:/icon/png图片/8.png</iconset>
   </property>
   <property name="text">
    <string>字体(&amp;Z)</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/png图片/9.png</normaloff>:/icon/png图片/9.png</iconset>
   </property>
   <property name="text">
    <string>颜色(&amp;C)</string>
   </property>
  </action>
  <action name="dateAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/png图片/11.png</normaloff>:/icon/png图片/11.png</iconset>
   </property>
   <property name="text">
    <string>日期与时间(&amp;D)</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/png图片/12.png</normaloff>:/icon/png图片/12.png</iconset>
   </property>
   <property name="text">
    <string> 撤销</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/png图片/a.jpg</normaloff>:/icon/png图片/a.jpg</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/png图片/b.jpg</normaloff>:/icon/png图片/b.jpg</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="pastAction">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="selectAllAction">
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="updateAction">
   <property name="text">
    <string>更新(&amp;U)</string>
   </property>
  </action>
  <action name="aboutSoftwareAction">
   <property name="text">
    <string>关于软件(&amp;S)</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>帮助(&amp;E)</string>
   </property>
  </action>
  <action name="aboutWebAction">
   <property name="text">
    <string>关于官网(&amp;W)</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>关于Qt(&amp;Q)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
